File: app/src/main/java/com/gee12/mytetroid/views/fragments/settings/SettingsFragment.java
package com.gee12.mytetroid.views.fragments.settings;

import android.os.Bundle;

import com.gee12.mytetroid.R;

public class SettingsFragment extends TetroidSettingsFragment {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        super.onCreatePreferences(savedInstanceState, rootKey);
        setPreferencesFromResource(R.xml.prefs, rootKey);

        getActivity().setTitle(R.string.title_settings);
    }
}

File: app/src/main/java/com/gee12/mytetroid/views/fragments/settings/SettingsFragment.kt
package com.gee12.mytetroid.views.fragments.settings

import android.os.Bundle
import com.gee12.mytetroid.R

class SettingsFragment : TetroidSettingsFragment() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        super.onCreatePreferences(savedInstanceState, rootKey)
        setPreferencesFromResource(R.xml.prefs, rootKey)
        requireActivity().setTitle(R.string.title_settings)
    }
}